packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/ValueExpression.cs

```diff
@@ -59,6 +60,18 @@ public InvokeInstanceMethodExpression Invoke(string methodName, IReadOnlyList<Va
 
         public CastExpression CastTo(CSharpType to) => new CastExpression(this, to);
 
+        public BoolSnippet GreaterThan(ValueExpression other) => new(new BinaryOperatorExpression(">", this, other));
+
+        public BoolSnippet LessThan(ValueExpression other) => new(new BinaryOperatorExpression("<", this, other));
+
+        public BoolSnippet Equal(ValueExpression other) => new(new BinaryOperatorExpression("==", this, other));
+
+        public BoolSnippet NotEqual(ValueExpression other) => new(new BinaryOperatorExpression("!=", this, other));
+
+        public BoolSnippet Is(ValueExpression other) => new(new BinaryOperatorExpression("is", this, other));
+
+        public  MethodBodyStatement Increment() => new UnaryOperatorExpression("++", this, true).Terminate();
```

Comment:
would it be OK if this is a MethodBodyStatement?
For instance, one of the scenario that this might be useful is some one wants to construct a `for` loop, but it should accepts three expressions.
How about we have two version? Such as:
```suggestion
        public ValueExpression Increment() => new UnaryOperatorExpression("++", this, true);
        public MethodBodyStatement IncrementStatement() => new UnaryOperatorExpression("++", this, true).Terminate();
      
```